src/sentry/sentry_metrics/indexer/mock.py

```diff
@@ -1,4 +1,5 @@
-from typing import Optional
+import itertools
+from typing import DefaultDict, Optional
```

Comment:
Shouldn't you import `defaultdict` from collections rather than this, since you are not using
it as typeinfo but you are instantiating it

Reply:
Whoops yes, this was an auto-import I did not pay attention to.